lib/views/YYComparisonGuide.js

```diff
@@ -56,10 +55,7 @@ class YYComparisonGuide extends GuideLayer {
                 .append('g')
                 .attr('class', 'axis_guide x_guide--full_length');
 
-            var yLabelMargin = 4;
-            guideLayer.drawXLineGuide(xGuideSelection, xGuideEnterSelection, chart, {
-                labelY: height - margins.bottom - yLabelMargin
-            });
```

Comment:
This hack is no longer needed since all UI graphs now have default Nugget styles (showing x axis labels etc.)